overlay.d/05rhcos/usr/sbin/gcp-routes.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
Do we still need this now that it lives in the MCO?

https://github.com/openshift/machine-config-operator/pull/2011

Reply:
This is just reflecting the state of the internal git repository; let's not try to *also* change it at the same time.

Reply:
I mean if we succeed at this, then we could do a PR to drop that script and have it tested by Prow launching a cluster in Azure!